trpc/router/mailbox/issueGroups.ts

```
@@ -0,0 +1,358 @@
+        .filter((c) => c !== null && c.mailboxId === ctx.mailbox.id),
+    };
+  }),
+
+  merge: mailboxProcedure
+    .input(
+      z.object({
+        sourceId: z.number(),
+        targetId: z.number(),
+      }),
+    )
+    .mutation(async ({ ctx, input }) => {
```

ESLint

[error] 'ctx' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)